src/sonic-bgpcfgd/tests/test_bbr.py

```diff
@@ -185,7 +185,40 @@ def test___init_8():
             "bbr": expected_bbr_entries,
         }
     }
-    __init_common(constants, "BBRMgr::Initialized and enabled. Default state: 'enabled'", None, expected_bbr_entries, "enabled")
+    __init_common(constants, "BBRMgr::Initialized and enabled from constants. Default state: 'enabled'", None, expected_bbr_entries, "enabled")
+
+@patch('bgpcfgd.managers_bbr.swsscommon.DBConnector')
+def test___init_with_config_db(mock_db_connector):
+    mock_db_connector_instance = MagicMock()
+    mock_db_connector.return_value = mock_db_connector_instance
+    mock_db_connector_instance.get_table.return_value = {'all': {'status': 'enabled'}}
```

Comment:
this case covers the scenario of the bbr enabled in config_db, what about the scenario of the bbr disabled in config_db